x-pack/test/api_integration/apis/ml/job_validation/validate.ts

```diff
@@ -142,6 +142,7 @@ export default ({ getService }: FtrProviderContext) => {
               },
             ],
             influencers: ['order_id'],
+            // summary_count_field_name: 'doc_count',
```

Comment:
commented code

Reply:
Fixed in https://github.com/elastic/kibana/pull/109080/commits/b8cff0b7aeda6ff54c2101e9a0ac8d800b7c00a4